### PR TITLE
Remove implicit dependencies

### PR DESCRIPTION
**Pull request by nikolai-laevskii**
**Description:**
Automated update + install of missing package

**Related issue:**
—

**Check list:**
- [ ] Mark if documentation changes are required.
- [ ] Mark if tests were added or updated to cover the changes.

### CODE REVIEW

File: .github/workflows/versions.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ jobs:
           [
             '20-v8-canary',
             '20.0.0-v8-canary',
-            '20.0.0-v8-canary20221103f7e2421e91'
+            '20.0.0-v8-canary20221101e50e45c9f8'
```
Comment (IvanZosimov) [inline .github/workflows/versions.yml:64]: What is the reason for this change?

**Review by dmitry-shibanov — APPROVED** [review]

**Review by marko-zivic-93 — APPROVED** [review]
